vtr_flow/scripts/run_vtr_flow.pl

```diff
@@ -1687,3 +1711,74 @@ sub calculate_relaxed_W {
 
     return $relaxed_W;
 }
+
+sub cmp_full_vs_incr_STA {
+	# loop indices
+	my @indices = (0..6);
+
+	# default output file names
+	my @default_output_names = (
+			$packing_file,
+			$placement_file,
+			$routing_file,
+			$setup_timing_report,
+			$hold_timing_report,
+			$unconstrained_setup_timing_report,
+			$unconstrained_hold_timing_report
+		);
+
+	# The full STA flow should have already been run
+    # directly rename the output files
+    foreach my $index (@indices) {
+    	system_with_timeout(
+            $move_exec, "move.out",
+            $timeout, $temp_dir,
+            $default_output_names[$index], 
+            "full_" . $default_output_names[$index]
+        );
+    }
+
+    my @incremental_vpr_args = @forwarded_vpr_args;
+    push(@incremental_vpr_args, ("--timing_update_type", "incremental"));
+
+    # run incremental STA flow
+    my $fixed_W_log_file = "vpr.out";
```

Comment:
We should use a unique log file name here, as this will overwrite the previous VPR run's log message in vpr.out. Perhaps something like `vpr.incr_sta.out`.

We generally try to keep all main tool log files around, since this is helpful when debugging a failure.